File: Assignment__34/Question5.c
#include<stdio.h>

void DisplayASCII(char ch)
{
    printf("Decimal     : %d\n", ch);
    printf("Octal       : %o\n", ch);
    printf("Hexadecimal : %X\n", ch);
}

int main()
{
    char ch;

    printf("\nEnter any character: ");
    scanf(" %c", &ch);

    DisplayASCII(ch);

    return 0;
}

File: Assignment__35/Question5.java
import java.util.Scanner;

class ASCII
{
    public void DisplayASCII(char ch)
    {
        System.out.println("Decimal     : " + (int)ch);
        System.out.println("Octal       : " + Integer.toOctalString(ch));
        System.out.println("Hexadecimal : " + Integer.toHexString(ch).toUpperCase());
    }
}

public class Question5
{
    public static void main(String[] args)
    {
        Scanner scan = new Scanner(System.in);

        System.out.println("Enter any character: ");
        char ch = scan.next().charAt(0);

        ASCII obj = new ASCII();
        obj.DisplayASCII(ch);
    }
}
